.github/workflows/python-publish.yml

```diff
@@ -14,18 +14,28 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
         python-version: '3.x'
+    
+    - name: Set up R (required for rpy2)
```

Comment:
R may not be required for the same reason. It does not interact with package install.